src/main/java/org/dcsa/tnt/Application.java

```diff
@@ -1,11 +1,17 @@
 package org.dcsa.tnt;
 
+import org.dcsa.core.repository.ExtendedRepositoryImpl;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
 
 @SpringBootApplication
+@ComponentScan("org.dcsa")
+@EnableR2dbcRepositories(basePackages = {"org.dcsa"}, repositoryBaseClass = ExtendedRepositoryImpl.class)
```

Comment:
These configurations have been moved to `ApplicationConfig`, not required here anymore.